include/z64pause.h

```diff
@@ -20,6 +20,33 @@ typedef enum {
     /* 0x04 */ PAUSE_WORLD_MAP
 } PauseMenuPage;
 
+// The XZ coordinates in which direction each pause page is at
+// e.g. the item page is in the -z direction
+/*
+ *      [   item   ]
+ *
+ *   [                [
+ *              x
+ * equip     o-->    map
+ *           |
+ *   ]     z v        ]
+ *
+ *      [   quest  ]
+ */
```

Comment:
I was thinking maybe something like this 
```c
/*
 *         [   item   ]
 *
 *                    
 *              x
 * [ equip ]    o-->    [ map ]
 *              |
 *            z v        
 *
 *         [   quest  ]
 */
 ```
 Im not sure if that formatting will look right so ill include an image as well
![image](https://github.com/zeldaret/oot/assets/22045047/08454cf8-e54c-41f6-8635-a91059e17123)

Reply:
programmers art moment
I was trying to mimic the direction in which each page stretches by keeping the [] above/below

maybe `<>v^` would be clearer for that purpose?

```
 *      <   item   >
 *
 *   ^                ^
 *              x
 * equip     o-->    map
 *           |
 *   v     z v        v
 *
 *      <   quest  >
 */
```

Or I'm ok `[linear]`ing it

Reply:
Sure that looks nice